Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class BleManager {
+  Future<void> createClient(String restoreStateIdentifier) async
+  Future<void> destroyClient() async
```

Comment:
It would be nice to construct API to allow multiple instances of CentralManager in the future. Actual implementation to allow that could be added later on.